src/index.js

```diff
@@ -25,6 +24,16 @@ const argv = yargs(hideBin(process.argv))
 		type: 'string',
 		description: 'ANT undername to update.',
 		default: '@',
+	})
+	.option('eth', {
+		alias: 'e',
+		type: 'boolean',
+		description: 'Connect with an ETH wallet instead of an Arweve wallet.',
+	})
+	.option('pol', {
```

Comment:
when i see `pol` i think polkadot, not polygon 🤔 `DOT` is the polkadot ticker, `MATIC` is the polygon ticker ( i think? )

I think we can just call these their full names, eg polygon, ethereum, solana - that way we avoid confusion and still have the alias available for short hand

Reply:
updated

Reply:
mmm its still using pol and eth?